kubernetes/polkassembly/values.yaml

```diff
@@ -151,3 +151,39 @@ chainDbWatcher:
   resources: {}
   tolerations: []
   affinity: {}
+
+healthMonitor:
+  name: health-monitor
+  replicaCount: 1
+
+  config:
+    # Listening Port
+    PORT: 8034
+    # polkassembly auth-server
+    AUTH_SERVER: "http://auth-server-service:8010"
+    # polkassembly hasura graphql service
+    HASURA_SERVER: "http://hasura-service:8080"
+    # polkassembly hasura chain-db service
+    CHAIN_DB_WATCHER_SERVER: "http://chain-db-watcher-service:8019"
+    # nomidot backend server ( CHAIN_DB_SERVER == CHAIN_DB_GRAPHQL_URL )
+    CHAIN_DB_SERVER: "http://nomidot-server.nomidot-staging:4000"
+    # nodewatcher server ( aka Prisma server )
+    CHAIN_DB: "http://nodedwatcher.nodewatcher-staging:4466"
```

Comment:
This server is in `staging` and the domain won't be accessible in prod I think will it? totally forgot if this one has any way to be accessed from inside the container.

Reply:
yes, i forgot as well. i will look into that.

Reply:
TBH I don't think it's super easy/relevant to bother too much about this one. if it's down, the `chain-db-server` (aka 4000) will also not answer. It'd be cool if we can tell which one is having problems, but if it's too annoying, then we'll do without.